docs/exact_infer/bp_wmp.rst

```diff
@@ -0,0 +1,5 @@
+Belief Propagation with Message Passing
+==================
```

Comment:
@tillwenke I think the `====` line needs to be at least as long as the text: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
done